src/hotel_reservations/mlflow_processor.py

```
@@ -176,13 +218,103 @@
 
     def get_model_version_by_tag(self, git_sha):
-        filter_string = f"name='{self.model_name}' and tags.tag='{git_sha}'"
-        model_version_by_tag = client.search_model_versions(filter_string)
+        filter_string = f"name='{self.full_model_path}' and tags.git_sha='{git_sha}'"
+        model_version_by_tag = mlflow_client.search_model_versions(filter_string)
 
         return model_version_by_tag
+
+    def create_online_table(self):
+        online_table_name = self.config["catalog_name"] + "." + self.config["schema_name"] + "." + "fe_online"
+        spec = OnlineTableSpec(
```

Ruff

`OnlineTableSpec` may be undefined, or defined from star imports

(F405)